SDK/RichRelevanceSDK/RCHRequestBuilder.h

```diff
@@ -88,13 +88,19 @@ OBJC_EXTERN NSString *const kRCHRequestBuilderDefaultDictKeyValueDelimiter;
  *
  *  @param value The new value to add, an instance of either NSString or NSNumber
  *  @param key   The key under which to add the value
- *
- *  @return <#return value description#>
  */
-- (instancetype)addValue:(id)value toArrayForhKey:(NSString *)key;
+- (instancetype)addValue:(id)value toArrayForKey:(NSString *)key;
+
+/*!
+ *  Add a new value to an array that is represented as multiple arguments, not a string delimited array.
+ *  @param value The new value to add, an instance of either NSString or NSNumber
+ *  @param key   The key under which to add the value
+ */
```

Comment:
#nth I don't remember if I did this for the others, but it might help to show an example of what kind of key/value pair query param this generates.